Makefile

```diff
@@ -519,7 +527,16 @@ release-binary: $(RELEASE_DIR)
 release-staging: ## Builds and push container images to the staging bucket.
 	REGISTRY=$(STAGING_REGISTRY) $(MAKE) docker-build-all docker-push-all release-alias-tag
 
-RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+.PHONY: release-staging-nightly
+release-staging-nightly: ## Builds and push container images to the staging bucket.
```

Comment:
is this comment still accurate? We're not actually building and pushing a new image anymore, right?

Reply:
Correct, fixed.